accounts-db/src/accounts_index/bucket_map_holder.rs

```diff
@@ -232,9 +298,37 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> BucketMapHolder<T, U>
             threads,
             _phantom: PhantomData,
             startup_stats: Arc::default(),
+            threshold_entries_per_bin,
+        }
+    }
+
+    fn calculate_target_entries_per_bin(entries_per_bin: usize) -> usize {
+        if entries_per_bin == 0 {
+            0
+        } else {
+            let entries_per_bin_log2 = entries_per_bin.ilog2();
+            (1usize << entries_per_bin_log2) * 7 / 8
```

Comment:
Can you add comments indicating why we're choosing this number as the target, please?